frontend/.env

```diff
@@ -4,4 +4,8 @@ REACT_APP_JIRA_URL=https://jira.server
 REACT_APP_MAX_NON_BILLABLE_TO_BILLABLE_RATIO=.4
 REACT_APP_MAX_NON_BILLABLE_TO_BILLABLE_CELL_RATIO=.3  # https://forum.opencraft.com/t/sustainability-dashboard/431/15
 REACT_APP_COMPANY_NAME=OpenCraft
-REACT_APP_ACCOUNT_STRIP_NAMES=[]  # If multiple names are provided, stripping is performed for each of them, honoring their order.
+# If multiple names are provided, stripping is performed for each of them, honoring their order.
+REACT_APP_ACCOUNT_STRIP_NAMES=[]
```

Comment:
Local tests were breaking, because the comment was being caught as a part of the `REACT_APP_ACCOUNT_STRIP_NAMES` env variable.